### ISSUE TITLE
Comments for databases, tables, columns

### ISSUE THREAD

**Issue by pma-import** [phpmyadmin/phpmyadmin#6179]
Hi everyone!

For 10 years the use of phpMyAdmin really want to see the comments to display database and table next to their name. Modifications looked not much. Please do. 
[[img src=img23.jpg alt=foobar]]

---
* **Original URL**: https://sourceforge.net/p/phpmyadmin/feature-requests/1534/
* **Original author**: gavrilow
* **Attachment**: http://sourceforge.net/p/phpmyadmin/feature-requests/1534/attachment/img23.jpg

**Comment by nijel** [comment]
Why is this actually a performance problem? The table comments are stored in same IS table as list of tables, so getting both should be matter of one SQL query...

**Comment by zaverichintan** [comment]
@nijel Agreed to your point, but it will require one more variable with comments, otherwise it is very useful thing to view comments on the database and tables .
